vcs/util/path.go

```diff
@@ -0,0 +1,10 @@
+package util
+
+// Rel strips the leading "/" prefix from the path string, effectively turning
+// an absolute path into one relative to the root directory.
+func Rel(path string) string {
+	if len(path) > 0 && path[0] == '/' {
+		return path[1:]
+	}
+	return path
```

Comment:
The 4 lines above are equivalent to `return strings.TrimPrefix(path, "/")`, is that correct?

If so, I think you should just use `strings.TrimPrefix` instead for better readability.

Whenever I see some snippet of Go code like the above, I assume it must be slightly different logic and I spend 20-30 seconds figuring out the nuance that makes it not exactly the same as `strings.TrimPrefix`, and if it turns out to be equivalent, I think "why not just use `strings.TrimPrefix`?"